core/src/main/java/org/elasticsearch/plugins/PluginManager.java

```diff
@@ -595,7 +610,8 @@ private static void displayHelp(String message) {
         SysOut.println("    -h, --help                        : Prints this help message");
         SysOut.newline();
         SysOut.println(" [*] Plugin name could be:");
-        SysOut.println("     elasticsearch/plugin/version for official elasticsearch plugins (download from download.elasticsearch.org)");
+        SysOut.println("     plugin for official elasticsearch plugins (download from download.elastic.co for elasticsearch >= 2.0)");
```

Comment:
maybe just write `elasticsearch-plugin-name     Elasticsearch 2.0 Core plugin` (fixes the awkward spacing problem)

Also the phrasing assumes that elasticsearch commercial plugins are not official somewhat? Not sure if its just me being non-native speaking.